src/main/java/com/epam/ta/reportportal/dao/WidgetContentRepositoryImpl.java

```diff
@@ -93,8 +93,18 @@ public OverallStatisticsContent overallStatisticsContent(Filter filter, Sort sor
 				.fetch());
 	}
 
+	private List<JTestItemTypeEnum> getItemTypeCriteria(boolean includeMethods) {
+		List<JTestItemTypeEnum> itemTypes = Lists.newArrayList(JTestItemTypeEnum.STEP);
```

Comment:
it is incorrect
what if there would be Scenario type?

Reply:
Widgets get data only for step level items